test/functorch/test_ops.py

```diff
@@ -1676,6 +1676,8 @@ def test():
                 # TODO: implement batching rule
                 xfail("_batch_norm_with_update"),
                 xfail("as_strided", "partial_views"),
+                # xfail("igamma"),  # Unexpected success?
+                # xfail("igammac"),  # Unexpected success?
```

Comment:
Why do we consider this a unexpected success? It looks like this PR made sure the corresponding vmap rule is registered.